### ISSUE TITLE
Dynamic Type Not Supported In Product Tags Page

### ISSUE THREAD

**Issue by JB184351** [woocommerce/woocommerce-ios#6598]
Dynamic Type Not Supported In Product Tags Page. When on this screen Dynamic Type should be supported if possible.

![image](https://user-images.githubusercontent.com/36495451/162245372-fa6eba2e-a233-4728-9c33-0d09abd63d45.png)
